### PR TITLE
New version: Resolume.Avenue version 7.21.0.37521

### PR DESCRIPTION
**Pull request by resolume-buildserver**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/161519)

### CODE REVIEW

File: manifests/r/Resolume/Avenue/7.21.0.37521/Resolume.Avenue.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Resolume.Avenue
+PackageVersion: 7.21.0.37521
+PackageLocale: en-US
+Publisher: Resolume
+PublisherUrl: https://resolume.com
+PublisherSupportUrl: https://resolume.com/support/
+PrivacyUrl: https://resolume.com/company/privacy
```
Comment (stephengillie) [inline manifests/r/Resolume/Avenue/7.21.0.37521/Resolume.Avenue.locale.en-US.yaml:10]:
```suggestion
# PrivacyUrl:
```

Comment (resolume-buildserver) [reply manifests/r/Resolume/Avenue/7.21.0.37521/Resolume.Avenue.locale.en-US.yaml]: Fixed privacy policy was moved with our new website but no auto redirect rule in place.

**Pull request closed without merging**